src/SFA.DAS.Roatp.CourseManagement.Web/Infrastructure/RouteNames.cs

```diff
@@ -12,5 +12,7 @@ public static class RouteNames
         public const string ViewStandardDetails = "view-standard-details";
         public const string GetCourseContactDetails = "GetCourseContactDetails";
         public const string PostCourseContactDetails = "PostCourseContactDetails";
+        public const string ConfirmRegulatedStandard = "ConfirmRegulatedStandard";
```

Comment:
This should have `Get` suffix. I have suggested this elsewhere so resolve the comment if already taken care of.